src/game/scenes/mainmenu/menu_video.c

```diff
@@ -128,6 +133,18 @@ component *menu_video_create(scene *s) {
     menu_attach(menu, label_create(&tconf, "VIDEO"));
     menu_attach(menu, filler_create());
 
+    // Renderer selector
+    component *renderer_selector =
+        textselector_create(&tconf, "API:", "Choose the video renderer API to use", renderer_toggled, local);
+    menu_attach(menu, renderer_selector);
+
+    // Add standard resolutions
+    const char *r_name;
+    for(int r = 0; r < video_get_renderer_count(); r++) {
+        video_get_renderer_info(r, &r_name, NULL);
+        textselector_add_option(renderer_selector, r_name);
+    }
```

Comment:
comment has been copy-pasted and is not correct.
pedantic: Why is r_name outside of the for-loop?